pyccel/ast/core.py

```diff
@@ -427,7 +350,7 @@ def _sympystr(self, printer):
         return '{0} as {1}'.format(sstr(self.name), sstr(self.target))
 
     def __eq__(self, string):
-        if isinstance(string, (str, Symbol)):
+        if isinstance(string, (str, PyccelSymbol)):
```

Comment:
`PyccelSymbol` is now a subclass of `str`

Reply:
ec38596